### PR TITLE
Issue #2297

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding Note for #2297

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-identity-verification.md [hunk 1]
```diff
@@ -72,3 +72,6 @@ The table shows the minimum requirements for each deployment.
 | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) |
 | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter |
 | Azure Account, optional for Azure MFA billing | Azure Account, optional for Azure MFA billing |
+
+>[!IMPORTANT]
+> On windows on-premises deployment, the Windows Server Domain Controller **must be** at least 2016 and once per domain.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-identity-verification.md]:
```suggestion
> Windows On-premises Deployments **requires** that the Windows Server Domain Controller is **at least** version 2016 and once per domain.
```
Sorry for not completing the edit, but I struggle too much to understand what "**once per domain**" refers to.
Does it mean that you need _at least_ 1 instance of the DC, or is it that you cannot have more than 1 per domain?

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-identity-verification.md (hunk 1 above)
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-identity-verification.md]:
```suggestion
>For Windows on-premises deployment, the Windows Server Domain Controller **must be** version 2016 or later, and must be deployed once per domain.
```

Comment (j0rt3g4) [reply windows/security/identity-protection/hello-for-business/hello-identity-verification.md]: Got it I've modified that.

**Comment by j0rt3g4** [comment]
@nenonix I'm pretty sure you will do your changes here! :P

File: windows/security/identity-protection/hello-for-business/hello-identity-verification.md [hunk 2]
```diff
@@ -72,3 +72,6 @@ The table shows the minimum requirements for each deployment.
 | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) |
 | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter |
 | Azure Account, optional for Azure MFA billing | Azure Account, optional for Azure MFA billing |
+
+>[!IMPORTANT]
+>For Windows on-premises deployment, at least one Windows Server Domain Controller 2016 is required for each domain in case you have several domains.
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-identity-verification.md]:
```suggestion
>For Windows on-premises deployment, if you have several domains, at least one Windows Server Domain Controller 2016 is required for each domain.
```

**Comment by e0i** [comment]
@mapalko Would you review this PR based on #2297 ? Thanks.

File: windows/security/identity-protection/hello-for-business/hello-identity-verification.md [hunk 3]
```diff
@@ -72,3 +72,6 @@ The table shows the minimum requirements for each deployment.
 | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) |
 | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter |
 | Azure Account, optional for Azure MFA billing | Azure Account, optional for Azure MFA billing |
+
+>[!IMPORTANT]
+>For Windows on-premises deployment, if you have several domains, at least one Windows Server Domain Controller 2016 is required for each domain.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-identity-verification.md]:
```suggestion
>For Windows on-premises deployment, if you have several domains, at least one Windows Server 2016 Domain Controller is required for each domain.
```
My suggestion is based on the fact that "Windows Server 2016" is a product name in itself, and that it is not necessarily a Domain Controller in every scenario, but in this case, it needs to be one.

**Comment by illfated** [comment]
OK, if I have understood you correctly, what you suggest is to change the note in a similar way to this,
```diff
>[!IMPORTANT]
- >For Windows on-premises deployment, if you have several domains, at least one Windows Server 2016 Domain Controller is required for each domain.
+ >If you have several domains, at least one Windows Server 2016 Domain Controller is required for each domain. 
```
... and after that, include a link to the adequate domain controller planning guide.

**Comment by j0rt3g4** [comment]
Updated @illfated @mapalko

File: windows/security/identity-protection/hello-for-business/hello-identity-verification.md [hunk 4]
```diff
@@ -72,3 +72,6 @@ The table shows the minimum requirements for each deployment.
 | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) |
 | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter |
 | Azure Account, optional for Azure MFA billing | Azure Account, optional for Azure MFA billing |
+
+>[!IMPORTANT]
+>For Windows-hello deployment, at least one Windows Server Domain Controller 2016 is required for each domain in case you have several domains. For more information, please visit this [planning guide](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-adequate-domain-controllers).
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-identity-verification.md]:
```suggestion
> For Windows-hello deployment, if you have several domains, at least one Windows Server Domain Controller 2016 is required for each domain. For more information, see the [planning guide](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-adequate-domain-controllers).
```

**Comment by mapalko** [comment]
Yeah, that is what I was thinking.

**Comment by j0rt3g4** [comment]
@nenonix  thank you
@mapalko ready 👍

**Comment by e0i** [comment]
@mapalko The PR author has followed up with suggested changes as well as additional copy edits. Please merge if everything is in order. Thank you.

**Pull request merged**